modules/issue_tracker/php/issuerowprovisioner.class.inc

```diff
@@ -96,8 +96,9 @@ class IssueRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisioner
             $module = &$modules[$mname];
         }
 
-        $cid = $row['centerId'];
+        $cids = isset($row['centerId']) ?
+            [$row['centerId']] : array_keys(\Utility::getSiteList(false));
```

Comment:
Normally I would remind you that the proper way of declaring an array is `array($row['centerId'])` but i'll let it pass this time